src/style.scss

```diff
@@ -1,4 +1,5 @@
-@import url("//fonts.googleapis.com/css?family=Open+Sans:400,700");
+@import "./custom.scss";
```

Comment:
I think you can use short-hand `@import 'custom`;`

`_variables` is more common for me to recognize it as bootstrap variables. `custom` sounds like your own customised CSS class names.